packages/genesys-spark-components/src/components/beta/gux-popover-list-beta/gux-popover-list.tsx

```diff
@@ -192,14 +199,25 @@ export class GuxPopoverList {
     }
   }
 
+  private focusFirstItemInPopupList(): void {
+    afterNextRenderTimeout(() => {
+      void this.listElement.guxFocusFirstItem();
+    });
+  }
+
   connectedCallback(): void {
     trackComponent(this.root, { variant: this.position });
   }
 
+  componentWillLoad(): void {
+    this.listElement = this.root.querySelector('gux-list');
```

Comment:
This can be called in `connectedCallback` I think.

Reply:
fixed